### PR TITLE
add: tab items to content panel

### PR DESCRIPTION
**Pull request by alsturm**

### CODE REVIEW

File: src/modules/header/components/header.css [hunk 1]
```diff
@@ -178,6 +178,9 @@
 .is-open {
   transform: translate(28em, 0);
 }
+.is-active {
+  color: rgba(255,255,255,1) !important;
```
Comment (thiloSchlemmer) [inline src/modules/header/components/header.css:182]: the difference to the non-active color is (on my monitor) difficult to perceive.

**Review by thiloSchlemmer — APPROVED** [review]
Looks good to me. Minor suggestion as inline-comment

File: test/modules/menu/components/contentPanel/ContentPanel.test.js [hunk 2]
```diff
@@ -123,6 +124,22 @@ describe('ContentPanelElement', () => {
...
+			expect(element.shadowRoot.querySelectorAll('.tabcontent')[0].style.display).toBe('block');
+			expect(element.shadowRoot.querySelectorAll('.tabcontent')[1].style.display).toBe('none');
+			expect(element.shadowRoot.querySelectorAll('.tabcontent')[2].style.display).toBe('none');
+			expect(element.shadowRoot.querySelectorAll('.tabcontent')[3].style.display).toBe('none');
+			expect(element.shadowRoot.querySelectorAll('.tabcontent')[4].style.display).toBe('none');
+
+		});
+
 
 	});
```
Comment (taulinger) [inline test/modules/menu/components/contentPanel/ContentPanel.test.js:144]: We also need  test(s) for changing the tab-index:
```
describe('when tab index changed', () => { ...
```

**Review by taulinger — COMMENTED** [review]
Very nice proceedings!
The styling of the layer-manager is done within this PR?

**Review by taulinger — APPROVED** [review]

**Pull request merged**